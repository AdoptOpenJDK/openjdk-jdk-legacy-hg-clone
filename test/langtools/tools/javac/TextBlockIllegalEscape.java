/*
 * @test /nodynamiccopyright/
 * @bug 8227640
 * @summary Verify that illegal escapes in text blocks do not crash the javac.
 * @compile/fail/ref=TextBlockIllegalEscape.out --enable-preview -source ${jdk.version} -XDrawDiagnostics TextBlockIllegalEscape.java
 */

public class TextBlockIllegalEscape {
    static void test() {
        EQ("""
           \!
           """, "");
    }
}
