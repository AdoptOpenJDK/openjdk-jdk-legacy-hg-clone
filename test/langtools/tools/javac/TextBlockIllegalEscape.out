TextBlockIllegalEscape.java:11:13: compiler.err.illegal.esc.char
- compiler.note.preview.filename: TextBlockIllegalEscape.java
- compiler.note.preview.recompile
1 error
